package.json

```diff
@@ -22,7 +22,7 @@
     "bellhop-iframe": "^3.2.0",
     "chai": "4.2.0",
     "commander": "^5.0.0",
-    "eslint": "^8.30.0",
+    "eslint": "^7.0.0",
```

Comment:
Is there a reason we lowered the version?

Reply:
Yes it was failing on > 8.0.0 - The file linter would need to be re-written to accommodate. I started down this path but then decided it made more sense to downgrade to 7.x rather than do a full rewrite, which seemed out of scope. Please let me know if you disagree and I should make 8.x work.

 Below is the result on eslint 8.x:
<img width="910" alt="Screenshot 2024-10-04 at 2 57 49 PM" src="https://github.com/user-attachments/assets/be89972a-50ad-4807-8afa-d62d07e20614">

Reply:
@902seanryan I revisited this today and made it work with eslint ^8.0.0!